### PR TITLE
Feature article

### PR DESCRIPTION
**Pull request by ilhamwibawa**
## Brief description

The article component consists of the article itself, a title and meta data.

## Developer Certificate of Origin

- [x] I certify that these changes according to the Developer Certificate of Origin 1.1 as described at <https://developercertificate.org/>.

## Further details

...

### CODE REVIEW

**Comment by rhyneav** [comment]
This is cool, really like the screenshot! Could you please change the base branch to develop?

**Comment by ilhamwibawa** [comment]
Hey @rhyneav , I have changed base branch to develop.

File: src/article.less [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	.article-meta {
+		font-size: 15px;
+		color: @muted;
+
+		a {
+			color: @muted-text;
+			background-image: none;
+
+			&:hover {
+				color: #000;
```
Comment (Fraham) [inline src/article.less]: This should be using a colour variable instead of the hard coded value

Comment (ilhamwibawa) [reply src/article.less]: thank for your feedback, I have changed color value to variable

**Comment by afzalsayed96** [comment]
Man your work is awesome!

**Comment by rhyneav** [comment]
Awesome, thank you for this addition!

**Pull request merged**